modules/afsocket/afinet-dest.c

```diff
@@ -362,6 +362,7 @@ afinet_dd_setup_addresses(AFSocketDestDriver *s)
   if (!_setup_dest_addr(self))
     return FALSE;
 
+  iv_invalidate_now();
```

Comment:
I think this is not necessary, as `_setup_dest_addr()` invalidates `now` through `resolve_hostname_to_sockaddr()`.